openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -128,6 +132,10 @@
 
             if (itemId in data) {
                 _.extend(self.oraData[i], data[itemId]);
+                if (self.oraData[i].staff_assessment) {
```

Comment:
Nit: Isn't `self.oraData[i]` the same as `oraItem`?  It seems a bit inconsistently used (or not used!) prior to the PR.